src/test/regress/expected/create_drop_database_propagation.out

```diff
@@ -932,15 +928,350 @@ DROP ROLE propagated_role, non_propagated_role;
+(3 rows)
+
+SELECT COUNT(*) FROM run_command_on_all_nodes('DROP DATABASE concurrent_create_db') WHERE success;
+ count
+---------------------------------------------------------------------
+     1
+(1 row)
+
+SELECT * FROM public.check_database_on_all_nodes('concurrent_create_db') ORDER BY node_type;
+      node_type       |                                                              result
+---------------------------------------------------------------------
+ coordinator (local)  | {"database_properties": null, "pg_dist_object_record_for_db_exists": false, "stale_pg_dist_object_record_for_a_db_exists": false}
+ worker node (remote) | {"database_properties": null, "pg_dist_object_record_for_db_exists": false, "stale_pg_dist_object_record_for_a_db_exists": false}
+ worker node (remote) | {"database_properties": null, "pg_dist_object_record_for_db_exists": false, "stale_pg_dist_object_record_for_a_db_exists": false}
+(3 rows)
+
+-- revert the system wide change that enables citus.enable_create_database_propagation on all nodes
+SELECT 1 FROM run_command_on_all_nodes('ALTER SYSTEM SET citus.enable_create_database_propagation TO OFF');
+ ?column?
+---------------------------------------------------------------------
+        1
+        1
+        1
+(3 rows)
+
+SELECT 1 FROM run_command_on_all_nodes('SELECT pg_reload_conf()');
+ ?column?
+---------------------------------------------------------------------
+        1
+        1
+        1
+(3 rows)
+
+SELECT pg_sleep(0.1);
+ pg_sleep
+---------------------------------------------------------------------
+
+(1 row)
+
+-- but keep it enabled for coordinator for the rest of the tests
+SET citus.enable_create_database_propagation TO ON;
+CREATE DATABASE distributed_db;
+CREATE USER no_createdb;
+SET ROLE no_createdb;
+SET citus.enable_create_database_propagation TO ON;
+CREATE DATABASE no_createdb;
+ERROR:  permission denied to create / rename database
+CONTEXT:  while executing command on localhost:xxxxx
+ALTER DATABASE distributed_db RENAME TO rename_test;
+ERROR:  permission denied to create / rename database
+CONTEXT:  while executing command on localhost:xxxxx
+DROP DATABASE distributed_db;
+ERROR:  must be owner of database distributed_db
+CONTEXT:  while executing command on localhost:xxxxx
+ALTER DATABASE distributed_db SET TABLESPACE pg_default;
+ERROR:  must be owner of database distributed_db
+CONTEXT:  while executing command on localhost:xxxxx
+ALTER DATABASE distributed_db SET timezone TO 'UTC';
+ERROR:  must be owner of database distributed_db
+CONTEXT:  while executing command on localhost:xxxxx
+ALTER DATABASE distributed_db RESET timezone;
+ERROR:  must be owner of database distributed_db
+CONTEXT:  while executing command on localhost:xxxxx
+GRANT ALL ON DATABASE distributed_db TO postgres;
+WARNING:  no privileges were granted for "distributed_db"
+RESET ROLE;
```

Comment:
super-nit: It would be nice if these errors did not come from another node, but instead the privileges were checked before sending commands there. Now users would get this "confusing" `CONTEXT:  while executing command on localhost:xxxxx`

Probably not important enough to do in this PR. So let's make an issue, seems like a "good first issue".

Reply:
let me quickly fix that in this PR while we're at it